code/modules/mining/equipment/survival_pod.dm

```diff
@@ -212,6 +212,14 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/door/window/survival_pod/left, 0)
 	density = TRUE
 	pixel_y = -32
 
+/obj/item/gps/computer/locator_tower
```

Comment:
I'd recommend putting singulo edit markers around this

Reply:
Done.